app/src/main/kotlin/compiler/intermediate_form/FunctionDependenciesAnalyzer.kt

```diff
@@ -16,11 +16,33 @@ import compiler.common.reference_collections.referenceSetOf
 import compiler.semantic_analysis.VariablePropertiesAnalyzer
 
 object FunctionDependenciesAnalyzer {
+    fun createUniqueIdentifiers(program: Program): ReferenceMap<Function, UniqueIdentifier> {
+        val uniqueIdentifiers = ReferenceHashMap<Function, UniqueIdentifier>()
+        val identifierFactory = UniqueIdentifierFactory()
+        fun nameFunction(function: Function, pathSoFar: String?): String? {
```

Comment:
```suggestion
        fun nameFunction(function: Function, pathSoFar: String?): String {
```
Returned value is never null.